### ISSUE TITLE
Docs not emitted for fields

### ISSUE THREAD

**Issue by Egorand** [square/wire#781]
In `java_interop` mode, field docs are added to `Builder` methods. In no-interop mode, docs are lost. They should be printed either on the properties, or in the class header.

**Comment by Egorand** [comment]
I'd like to have something like this:
```kotlin
/**
 * @property name The customer's full name.
 * @property id The customer's ID number.
 * @property email Email address for the customer.
 * @property phone A list of the customer's phone numbers.
 */
data class Person(
    val name: String,
    val id: Int,
    val email: String? = null,
    val phone: List<PhoneNumber> = emptyList(),
    val unknownFields: ByteString = ByteString.EMPTY
) : Message<Person, Person.Builder>(ADAPTER, unknownFields) {
```
But KotlinPoet currently has no support for `@property` KDoc tags.

**Comment by oldergod** [comment]
https://github.com/square/kotlinpoet/pull/569

**Comment by Egorand** [comment]
KP PR has been merged

**Comment by Egorand** [comment]
Discussed this a while ago with @jrodbx, an alternative proposal is to go with this:
```
data class Person(
    /**
     * The customer's full name.
     */
    val name: String,
    /**
     * The customer's ID number.
     */
    val id: Int,
    /**
     * Email address for the customer.
     */
    val email: String? = null,
    /**
     * A list of the customer's phone numbers.
     */
    val phone: List<PhoneNumber> = emptyList(),
    val unknownFields: ByteString = ByteString.EMPTY
) : Message<Person, Person.Builder>(ADAPTER, unknownFields) {
```
This is not currently supported in KotlinPoet.

**Comment by Egorand** [comment]
Closed with #840
